source/documentation/administration_guide/topics/Setting_Password.adoc

```diff
@@ -2,7 +2,11 @@
 [id="setting-a-user-password_{context}"]
 = Setting a User Password
 
-You can create a password. You must set a value for `--password-valid-to`, otherwise the password expiry time defaults to the current time. The date format is `yyyy-MM-dd HH:mm:ssX`. In this example, `-0800` stands for GMT minus 8 hours. For more options, run `ovirt-aaa-jdbc-tool user password-reset --help`.
+You can create a password. You must set a value for `--password-valid-to`, otherwise the password expiry time defaults to the current time.
++
+The date format is `yyyy-MM-dd HH:mm:ssX`. Where `X` is the time zone offset from UTC. In this example, `-0800` stands for GMT minus 8 hours. For zero offset, use the value `Z`.
```

Comment:
Consider: The time format is `YYYY-MM-DD hh:mm:ssX` -- IBM describes timestamps as:
"Sometimes, displaying a date and time as an all-numeric timestamp enhances readability. If you decide to use a timestamp, use one that follows ISO 8601, but that does not specify a "T" delimiter between the date and time, and that specifies seconds. Such a timestamp has the form YYYY-MM-DD hh:mm:ss.n..., where seconds and fractions of a second are optional."